generators/app/templates/src/app/components/Suspense/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Suspense } from 'react';
+
+import Loading from '../Spinner/components/loading';
+
+interface Props {
+  fallback?: React.ElementType;
+  children: React.ReactNode;
+}
+
+function CustomSuspense({ fallback = Loading, children }: Props) {
```

Comment:
If fallback is optional then the default value isn't needed. If it is, then remove the optional marker.

Reply:
This doesn't work in TS I think. If you define it as not optional, the user must pass something to match the interface. If that's the case, I'd leave it as it is

Reply:
@pabloferro can you confirm this?

Reply:
Capi is right.
We should provide default values to optional props or handle the undefined case.